Hastebin crashes: document not found key=index

Yesterday, my hastebin instance seems to have crashed with the following error:
```
warn: document not found key=index
/node_modules/memcached/lib/utils.js:152
  return value.replace(/(\r|\n)/g, '\\$1');
               ^

TypeError: Cannot read property 'replace' of undefined
    at Object.exports.escapeValue (/var/www/haste.my.domain/node_modules/memcached/lib/utils.js:152:16)
    at Client.setters (/var/www/haste.my.domain/node_modules/memcached/lib/memcached.js:929:19)
    at Client.bowlofcurry [as set] (/var/www/haste.my.domain/node_modules/memcached/lib/utils.js:126:15)
    at MemcachedDocumentStore.set (/var/www/haste.my.domain/lib/document_stores/memcached.js:29:17)
    at Object.client.get [as callback] (/var/www/haste.my.domain/lib/document_stores/memcached.js:41:14)
    at Client.memcached.delegateCallback (/var/www/haste.my.domain/node_modules/memcached/lib/memcached.js:687:12)
    at Client.rawDataReceived (/var/www/haste.my.domain/node_modules/memcached/lib/memcached.js:758:22)
    at Client.BufferBuffer (/var/www/haste.my.domain/node_modules/memcached/lib/memcached.js:678:12)
    at Socket.bowlofcurry (/var/www/haste.my.domain/node_modules/memcached/lib/utils.js:126:15)
    at Socket.emit (events.js:188:13)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! haste@0.1.0 start: `node server.js`
npm ERR! Exit status 1
```

I use the following storage settings:
```
  "storage": {
    "type": "memcached",
    "host": "127.0.0.1",
    "port": 11211,
    "expire": 2592000
  },
```
When I restarted it, it worked like a charm. Any reason for this?

--- a/lib/document_stores/memcached.js
+++ b/lib/document_stores/memcached.js
@@ -26,7 +26,7 @@ class MemcachedDocumentStore {
 
   // Save file in a key
   set(key, data, callback, skipExpire) {
-    this.client.set(key, data, skipExpire ? 0 : this.expire, (error) => {
+    this.client.set(key, data, skipExpire ? 0 : this.expire || 0, (error) => {
       callback(!error);
     });
   }
@@ -34,10 +34,12 @@ class MemcachedDocumentStore {
   // Get a file from a key
   get(key, callback, skipExpire) {
     this.client.get(key, (error, data) => {
-      callback(error ? false : data);
+      const value = error ? false : data;
+
+      callback(value);
 
       // Update the key so that the expiration is pushed forward
-      if (!skipExpire) {
+      if (value && !skipExpire) {
         this.set(key, data, (updateSucceeded) => {
           if (!updateSucceeded) {
             winston.error('failed to update expiration on GET', {key});

